src/USD.NET/USD.NET.csproj

```diff
@@ -256,6 +256,9 @@
     <Compile Include="generated\pxr\usd\sdf\SdfStringListOpVector.cs" />
```

Comment:
A quick "improvement" could be to only include something like "generated\pxr\**\*.cs" once instead of listing each csharp file generated. That would eventually avoid seeing this file in future diff.

Reply:
Can you do that in VS? As soon as you add a folder all the filepaths are added to the csproj.

Reply:
I did that on my side, it worked well. Do what you think it's best and works. After a second thought, maybe we prefer to have this explicit diff of new files and the one removed even if it's a bit big.